src/main/java/com/codejoust/main/model/report/SubmissionGroupReport.java

```diff
@@ -0,0 +1,42 @@
+package com.codejoust.main.model.report;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+
+import com.codejoust.main.model.User;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Entity
+@Getter
+@Setter
+public class SubmissionGroupReport {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+
+    private String gameReportId;
+
+    @OneToMany(mappedBy = "submissionGroupReport", fetch = FetchType.EAGER)
+    @Setter(AccessLevel.PRIVATE)
+    private List<SubmissionReport> submissionReports = new ArrayList<>();
+
+    // This column holds the primary key of the user who made these submissions
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "user_table_id")
+    private User user;
```

Comment:
I _very briefly_ looked through that SO post so this may be way off base, but would it work to simply move the JoinColumn to the OneToMany side (as in the first answer)?

Reply:
It turns out this works great - the reason it wasn't working originally is because I didn't include `@Fetch(value = FetchMode.SUBSELECT)` on the `OneToMany` side, so I mistakenly thought that this wasn't possible. I see that it works great now though, so I'll update the data model associations to remove the unnecessary `ManyToOne` annotations and fields.